README

```diff
@@ -230,6 +230,18 @@ SUSE:
 Alternatively the MPI requirement can be met by using OpenHPC
 packages.
 
+If power governing or power balancing is the intended use case
+for GEOPM deployment then there is an additional dependency
+on the system being configured to support RAPL execute (on a compute node):
```

Comment:
s/system/BIOS

Reply:
...then there is an additional dependency on the BIOS being configured to support RAPL control.  To check for BIOS support, execute the following on a compute node: